test/tools/Modules/HttpListener/HttpListener.psm1

```diff
@@ -134,6 +134,13 @@ Function Start-HTTPListener {
                             $contentType = $queryItems["contenttype"]
                             $output = $queryItems["output"]
                         }
+
+                        # Echo the request as the output.
+                        "echo"
```

Comment:
"echo" doesn't seem to be necessary as "response" basically does the same thing.  I would suggest updating "response" if you want json (perhaps based on "contenttype"